### PR TITLE
Task 312-315: EuroParl (Swedish and English)

### PR DESCRIPTION
**Pull request by Mirzyaaliii**
task312_EuroParl_sv_en_translation  -->  Given a Swedish sentence, convert it into English.
task313_EuroParl_en_sv_translation  -->  Given a Swedish sentence, convert it into English.
task314_EuroParl_sv-en_classification  -->  Given a Swedish sentence and its corresponding English, classify whether it is correct or not by providing 'Yes' or 'No'
task315_EuroParl_sv-en_language_identificcation  -->  Given a sentence, identify its language whether it is in Swedish or English.

Please review.

### CODE REVIEW

**Comment by swarooprm** [comment]
@shailaja183 can you review these tasks?

**Comment by shailaja183** [comment]
Hi @Mirzyaaliii, looks good mostly, few minor comments

Task 312,313,315
1. Please write full language name (eg. English) in fields Input_language, Output_language and instruction_language instead of abbreviations (eg. en) 

Task 312-313
2. Please make Definition a bit more detailed on what aspects of translation should be taken care of (upper/lowercase, numbers, proper nouns, punctuations etc.). See tasks 250-253 for example. 
3. For Negative Examples, write detailed justification about what aspect of translation is wrong. See tasks 250-253 for example. 

Task 314
4. Definition looks a little convoluted. Break down Definition into multiple sentences. Also simplify the statement "your task is to classify whether the translation from Swedish to the English is correct or not by providing 'Yes' and 'No'.". Remove 'the' before English.
5. In Positive Examples with output "No", provide detailed justification- why the target sentence is not a good translation of the source sentence.

Task 315
6. "there is no third language in input sentence." is repetition of "Input sentence can be in Swedish or English language only", remove either. 
7. In explanation of negative examples, is "Input language" a typo? Should it be "Input sentence"? (let me know if I misunderstood). Also, Instead of "Input sentence is in" use "Input sentence can be" would probably make more sense.

**Comment by Mirzyaaliii** [comment]
> Hi @Mirzyaaliii, looks good mostly, few minor comments
> 
> Task 312,313,315
> 
> 1. Please write full language name (eg. English) in fields Input_language, Output_language and instruction_language instead of abbreviations (eg. en)
> 
> Task 312-313
> 2. Please make Definition a bit more detailed on what aspects of translation should be taken care of (upper/lowercase, numbers, proper nouns, punctuations etc.). See tasks 250-253 for example.
> 3. For Negative Examples, write detailed justification about what aspect of translation is wrong. See tasks 250-253 for example.
> 
> Task 314
> 4. Definition looks a little convoluted. Break down Definition into multiple sentences. Also simplify the statement "your task is to classify whether the translation from Swedish to the English is correct or not by providing 'Yes' and 'No'.". Remove 'the' before English.
> 5. In Positive Examples with output "No", provide detailed justification- why the target sentence is not a good translation of the source sentence.
> 
> Task 315
> 6. "there is no third language in input sentence." is repetition of "Input sentence can be in Swedish or English language only", remove either.
> 7. In explanation of negative examples, is "Input language" a typo? Should it be "Input sentence"? (let me know if I misunderstood). Also, Instead of "Input sentence is in" use "Input sentence can be" would probably make more sense.

Updated all the tasks, please review again.

**Comment by shailaja183** [comment]
Looks good to me. @swarooprm give a final check.

File: tasks/README.md [hunk 1]
```diff
@@ -262,10 +262,17 @@ Name | Summary | Category
...
+=======
 `task282_scruples_event_time` | Given an anecdotes, find whether it has already happened or it may happen in the future  | Answer Generation
 `task287_casehold_legal_incorrect_answer_generation` | Given a prompt from a judicial decision and multiple potential holdings, choose one of the incorrect options | Legal , Incorrect Answer Generation
 `task290_TELLMYWHY_question_answerability` | Given a story and a question, decide whether or not the question is answerable | Classification
 `task301_record_question_generation` | Given a passage, generate a fill-in-the-gap question based on it | Question Generation
 `task302_record_classification` | Given a passage and a question, classify the answer to the question based on the options | Classification
 `task303_record_incorrect_answer_generation` | Given a passage and a question, write an incorrect answer for the question | Incorrect Answer Generation
 `task339_record_answer_generation` | Given a passage and a question, answer the question based on the passage | Answer Generation
+>>>>>>> master
```
Comment (danyaljj) [inline tasks/README.md]: Fix the merge issues in this file.

Comment (Mirzyaaliii) [reply tasks/README.md]: Done!

File: tasks/README.md [hunk 2]
```diff
@@ -260,6 +260,10 @@ Name | Summary | Category
 `task274_overruling_legal_classification`	| Given a sentence, classify it into overruling or non-overruling| Legal , Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
 `task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task312_EuroParl_sv_en_translation` | Given a Swedish sentence, convert it into English. | Translation 
+`task313_EuroParl_en_sv_translation` | Given a Swedish sentence, convert it into English. | Translation
+`task314_EuroParl_sv-en_classification` | Given a Swedish sentence and its corresponding English, classify whether it is correct or not by providing 'Yes' or 'No' | Classification
+`task315_EuroParl_sv-en_language_identificcation` | Given a sentence, identify its language whether it is in Swedish or English. | Language Identification (Binary)
```
Comment (danyaljj) [inline tasks/README.md]: Let's sort the rows based on their task ids.

Comment (Mirzyaaliii) [reply tasks/README.md]: Done!

**Comment by danyaljj** [comment]
Thanks to both of you!

**Pull request merged**